Tests/Context/ContextTest.php

```diff
@@ -98,4 +98,17 @@ public function testSerializeNull()
 
         $this->assertEquals(true, $this->context->getSerializeNull());
     }
+
+    public function testExclusionStrategy()
+    {
+        $strategy1 = $this->getMock('JMS\Serializer\Exclusion\ExclusionStrategyInterface');
+        $strategy2 = $this->getMock('JMS\Serializer\Exclusion\ExclusionStrategyInterface');
```

Comment:
This is deprecated :/ you can replace it by `getMockBuilder(...)->getMock()` and also use the `::class` constant.